src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs

```diff
@@ -47,35 +47,36 @@ public static AadIssuerValidator GetIssuerValidator(string aadAuthority)
             if (string.IsNullOrEmpty(aadAuthority))
                 throw new ArgumentNullException(nameof(aadAuthority));
 
-            if (s_issuerValidators.TryGetValue(aadAuthority, out AadIssuerValidator aadIssuerValidator))
+            string authority;
+            try
             {
-                return aadIssuerValidator;
+                authority = new Uri(aadAuthority).Authority;
```

Comment:
Let's be careful. the OIDC authority contains paths (the instance does not), whereas the Uri.Authority is really the instance. I'm not a big fan of naming the instance `authority`. @jennyf19, @bgavrilMS what do you think?

Reply:
Hmm, yea it's an overloaded term. Uri.Authority is host + port. Could rename it to authorityHost?